# To be BSI basic security compliant, containers need to run with a read-only root file system.
# To configure it's logging, the UDM REST API calls the `log_init` function from the file patched below:
# `log_init('/dev/stdout', args.debug, args.processes != 1)`
#
# This function executes a change owner operation which leads to an error if paired with a read-only file system.
# Until it is fixed upstream, the below lines need to be deleted from the file.

--- /usr/lib/python3/dist-packages/univention/management/console/log.py	2024-01-31 14:49:51.000000000 +0100
+++ /usr/lib/python3/dist-packages/univention/management/console/log.py	2024-02-01 15:23:22.601718519 +0100
@@ -88,9 +88,6 @@
         univention_debug_function=False,
         univention_debug_categories=('MAIN', 'LDAP', 'NETWORK', 'SSL', 'ADMIN', 'MODULE', 'AUTH', 'PARSER', 'LOCALE', 'ACL', 'RESOURCES', 'PROTOCOL'),
     )
-    adm = grp.getgrnam('adm')
-    os.fchown(fd.fileno(), 0, adm.gr_gid)
-    os.fchmod(fd.fileno(), 0o640)
     CORE.root.removeHandler(fallbackLoggingHandler)

     return fd
